src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/utils/TaskTestUtils.java

```diff
@@ -42,8 +44,16 @@ public class TaskTestUtils {
 
     private final CFTTaskDatabaseService cftTaskDatabaseService;
 
+    private final String profile;
+
     public TaskTestUtils(CFTTaskDatabaseService cftTaskDatabaseService) {
         this.cftTaskDatabaseService = cftTaskDatabaseService;
+        this.profile = "primary";
+    }
+
+    public TaskTestUtils(CFTTaskDatabaseService cftTaskDatabaseService, String profile) {
```

Comment:
It would make sense to keep only this constructor , and update the tests that use the old one